news/2025/2025-05-12-monthly-beatmapping-contest-may-2025.md

```diff
@@ -0,0 +1,160 @@
+
+This contest was themed around **finishers**. [See the full results here!](https://mappersguild.com/contests/results?contest=679299dbe655d9cc6466e3d0)
+
+#### Summary
+
+| Placement | User | Beatmap | Badge prize | [Contest points](/wiki/Contests/Contest_points) |
+| :-- | :-- | :-- | :-- | :-- |
+| Winner | [Alchyr](https://osu.ppy.sh/users/4993032) | [Akiri - Symphony Of The Lost Souls](https://osu.ppy.sh/beatmapsets/2365975) | ![](https://assets.ppy.sh/profile-badges/mbc-2020-2.png) | +2 (4 total) |
+| Runner-up | [tasuke912](https://osu.ppy.sh/users/2774767) | [Chroma - Destroy, Destroy, Destroy The Happy End](https://osu.ppy.sh/beatmapsets/2367066) | ![](https://assets.ppy.sh/profile-badges/mbc-2020-1.png) | +1 (4 total) |
+
+#### Top submissions
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/L7SMKpHMjvM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Send some appreciation to [BlackBN](https://osu.ppy.sh/users/6291741), [arcpotato](https://osu.ppy.sh/users/12842392), [meiqth](https://osu.ppy.sh/users/12565402), and [Yasuho](https://osu.ppy.sh/users/8458835) for judging this contest!
+
+## Game modes
+
+This month's contest is open to **osu!** and **osu!taiko** mappers.
+
+- [osu!catch contest listing](https://osu.ppy.sh/community/contests/246)
+- [osu!mania contest listing](https://osu.ppy.sh/community/contests/247)
+
+## Timeline
+
+- **[Beatmapping](#beatmapping) (3 weeks):** Contest is announced and beatmappers create their entries.
+- **[Screening](/wiki/Contests/Monthly_Beatmapping_Contest#screening) (1.5 weeks):** Entries are evaluated by experienced beatmappers. If there aren't many entries, this will be skipped in favour of a longer judging period.
+- **[Judging](/wiki/Contests/Monthly_Beatmapping_Contest#judging) (1.5 weeks):** Top entries from screening are judged in detail by a different crew of experienced beatmappers.
+- **[Rewards](#rewards) (~2 months from now):** Winners are announced at the start of another MBC.
+
+## Beatmapping
+
+### Limitation
+
+Limitation, restriction, gimmick, concept — what you call it doesn't matter. This is a contest-specific mapping rule that all submissions must abide by, ensuring that this month's contest is different from last month's, and the one before that, and the one before that, and... *(message truncated)*
+
+If you're confused about how these work, ask for clarification in the comments, or send a message to [pishifat](https://osu.ppy.sh/community/chat?sendto=3178418) (for osu!) or [Hivie](https://osu.ppy.sh/community/chat?sendto=14102976) (for osu!taiko).
+
+#### osu!
+
+According to the [ancient mapping lore](/wiki/Beatmapping/Mapping_techniques/Basics#jumps), mappers in ye olde days would sometimes use *new combos* to indicate *jumps*. In the current mapping climate, jumps are used everywhere, so that concept has been thrown out the window... but the MBC is here to bring it back!
```

Comment:
yes

```suggestion
According to the [ancient mapping lore](/wiki/Beatmapping/Mapping_techniques/Basics#jumps), mappers in ye olden days would sometimes use *new combos* to indicate *jumps*. In the current mapping climate, jumps are used everywhere, so that concept has been thrown out the window... but the MBC is here to bring it back!
```